benchmark-integration/jetty-perftest/jetty-perftest.gradle

```diff
@@ -10,6 +10,6 @@ jar {
   manifest {
     attributes(
       "Main-Class": "io.opentelemetry.perftest.jetty.JettyPerftest"
-    )
+      )
```

Comment:
Intended?

Reply:
unfortunate side effect of the new formatting.  If you know of better settings to make it better align with how intellij does it, please let me know.

Reply:
reverted auto-formatting of gradle files for now